core/lib/presentation/views/html_viewer/html_content_viewer_on_web_widget.dart

```diff
@@ -106,7 +110,7 @@ class _HtmlContentViewerOnWebState extends State<HtmlContentViewerOnWeb> {
             return false;  
           }
         
-          return url.protocol === "http:" || url.protocol === "https:";
+          return url.protocol === "http:" || url.protocol === "https:" || url.protocol === "mailto:";
```

Comment:
IMO you should change the method name to: `isValidUrl`

Reply:
ok